src/ci/resources/enable_metadata_archiving.inc.conf

```diff
@@ -21,6 +21,9 @@ services.MetadataService {
       # How long to require after workflow completion before going ahead with archiving:
       archive-delay = 10 minutes
 
+      # How often to send instrumentation metrics (eg. workflows left to archive metric)
+      instrumentation-interval = 1 minute
```

Comment:
TOL2: do we want to make a `firecloud-develop` PR to set this to something explicit in the live envs?

Reply:
Yes. I was planning to let BT know about this config when releasing Cromwell 62 to Terra. But now I am curious, if I add this config value to Cromwell config in firecloud-develop before Cromwell 62 is released, will it break things or will Cromwell simply ignore this config?

Reply:
It will just get ignored if nothing is looking for it yet

Reply:
That's great. I will create a associated firecloud-develop PR 👍